tailwind.config.js

```diff
@@ -93,24 +74,134 @@ module.exports = {
       'xl': '1440px'
     },
     textColor: theme => ({
-      "color-1": theme("colors.text.1"),
-      "color-2": theme("colors.text.2"),
-      "color-3": theme("colors.text.3"),
-      blue: {
-        1: theme("colors.blue.1"),
-        2: theme("colors.blue.2"),
-        3: theme("colors.blue.3"),
-      },
-      white: theme("colors.background")
+      color: {
+        "1": theme("colors.text.1"),
+        "2": theme("colors.text.2"),
+        "3": theme("colors.text.3"),
+        "link": theme("colors.text.link"),
+        "inverse": theme("colors.text.inverse")
+      }
     }),
-    transitionDuration: {
-      75: "75ms",
-      150: "150ms",
-      300: "300ms",
-      500: "500ms"
-    },
-    transitionTimingFunction: "cubic-bezier(0.215, 0.440, 0.420, 0.880)"
+    backgroundColor: theme => theme("colors.background"),
+    extend: {
+      animation: {
+        "in": "in 300ms ease-in-out",
+        "in-down": "in-down 300ms ease-in-out",
+        "in-up": "in-up 300ms ease-in-out",
+        "in-scale": "in-scale 300ms linear"
+      },
+      borderRadius: {
+        half: "50%"
+      },
+      boxShadow: {
+        0: "0 4px 8px -1px rgba(0, 0, 0, 0.08), 0 2px 4px -1px rgba(0, 0, 0, 0.04)",
+        1: "0 4px 8px -1px rgba(0, 0, 0, 0.08), 0 2px 4px -1px rgba(0, 0, 0, 0.04)",
+        "1-lg": "0 4px 16px 0 rgba(0, 0, 0, 0.08), 0 2px 8px 0 rgba(0, 0, 0, 0.04)",
+        "1-sm": "0 1px 6px -1px rgba(0, 0, 0, 0.16), 0 1px 2px -1px rgba(0, 0, 0, 0.08)",
+        2: "0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08)",
+        "2-lg": "0 12px 32px -2px rgba(0, 0, 0, 0.1), 0 4px 20px 0 rgba(0, 0, 0, 0.08)",
+        "2-sm": "0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16)",
+        'border-bottom': '0 1px 0 var(--calcite-ui-border-3)',
+        'outline-active': '0 0 0 1px var(--calcite-ui-brand)', 
+        'none': 'none',
+      },
+      keyframes: {
+        "in": {
+          "0%": {
+            opacity: 0
+          },
+          "100%": {
+            opacity: 1
+          }
+        },
+        "in-down": {
+          "0%": {
+            opacity: 0,
+            transform: "translate3D(0, -5px, 0)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "translate3D(0, 0, 0)"
+          }
+        },
+        "in-up": {
+          "0%": {
+            opacity: 0,
+            transform: "translate3D(0, 5px, 0)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "translate3D(0, 0, 0)"
+          }
+        },
+        "in-scale": {
+          "0%": {
+            opacity: 0,
+            transform: "scale3D(0.95, 0.95, 1)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "scale3D(1, 1, 1)"
+          }
+        }
+      },
+      opacity: {
+        disabled: "0.5"
+      },
+      spacing: {
+        "0.5": "0.125rem",
+        "2.5": "0.625rem",
```

Comment:
These are in the default tailwind scale, right?

Reply:
@paulcpederson You are correct. But we're currently on the older version that doesn't have those scales. There also isn't a consensus as to whether we want all the scale increments yet.

If we want to update to the newer Tailwind, I'm cool with nixing that extension.

Reply:
@paulcpederson Looks like updating to Tailwind 2.0 causes a bunch of stuff to break. But once we update and fix all the things, we should be able to remove these.